packages/react/src/ActionList/ActionList.features.stories.tsx

```diff
@@ -710,3 +712,25 @@ export const GroupWithFilledTitle = () => {
     </ActionList>
   )
 }
+
+export const ActionListWithButtonSemantics = () => {
```

Comment:
I wonder if we can enable the flag in all our stories and see how the changes are working?

I haven't tried it but maybe wrapping the story [here](https://github.com/primer/react/blob/main/packages/react/.storybook/preview.js#L215) with the feature flag component? 🤔 

```js
<BaseStyles>
  <FeatureFlags flags={{primer_react_action_list_item_as_button: true}}>
    <Story {...context} />
    {context.globals.colorScheme === 'all' && <p className="theme-name">{theme}</p>}
  </FeatureFlags>
</BaseStyles>
```

Reply:
We definitely could! But would we want to keep this on even after we ship this PR, or would it just during this PR?

Reply:
Sorry for the late response! I was thinking enabling the ff in this PR to make sure the changes are working as expected before releasing it to dotcom. It is like testing it in the small surface area (primer/react) and once we confirm that the changes are good, we can release. 

This is only a suggestion though. If you think we can directly release it that is okay too!

Reply:
Added! Should now be enabled for all stories!